tests/test-http-client-transport/src/test/java/org/eclipse/jetty/http/client/AsyncIOServletTest.java

```diff
@@ -1854,4 +1866,35 @@ public void stopServer() throws Exception
             super.stopServer();
         }
     }
+
+    static class Await
```

Comment:
Should this go in some kind of central test jar for all of our modules to use?

Perhaps we can extend build-resources for something like this.

Reply:
I also have no problem with making a new (independently versioned) jar in jetty-toolchain for these kinds of things.

Reply:
Since we already have the jetty-toolchain used in our testing, I'd prefer adding these kind of simple utility classes to that rather than bringing in a new dependency.

Reply:
I like the Await class. Might be able to use it in some of the session tests. Should it just go into jetty-util?

Reply:
Since there seems to be some interest in what the `Await` class does, I've decided to get rid of it and add a test dependency on awaitility as `Await` was just a cut-down version of that lib that was so far used only once.